### DISCUSSION TITLE
How to raise the 50 Mb limit for datasets in a locally run docker instance?

### DISCUSSION THREAD

**Discussion by dhdeangelis** [INSPIRE-MIF/helpdesk-validator#955] [Q&A] [answered]
The validator has a built in limit of 50 Mb for testing datasets. Is it possible to raise this limit for a locally run docker instance? In that case, how?

**Comment by jenriquesoriano** [comment, marked as answer]
Dear @dhdeangelis ,

it is possible to change the limit for a local docker instance configuring the following files:

- WEB-INF/classes/etf-config.properties, in line 34 you can change the etf.max.upload.size property.
- /res/nginx.conf, in line 3 you can modify the client_max_body_size property that is by default set at 50M (50MB).

Finally, when you start the docker a folder named etf is created on your home directory, if you have one already created, delete it before you build the new image.

Please let us know if further feedback is needed.

Best regards,
